sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -258,6 +260,25 @@ void DistortionCorrector<T>::undistortPointCloud(
     // Undistort a single point based on the strategy
     undistortPoint(it_x, it_y, it_z, it_twist, it_imu, time_offset, is_twist_valid, is_imu_valid);
 
+    if (!sensor_azimuth_coordinate.empty() && pointcloudTransformNeeded()) {
+      // Input frame should be in the sensor frame.
+      *it_distance = sqrt(*it_x * *it_x + *it_y * *it_y + *it_z * *it_z);
+      float cartesian_coordinate_azimuth = cv::fastAtan2(*it_y, *it_x);
```

Comment:
Here’s the translation:

I'd like to avoid introducing a dependency on OpenCV just for this function. Do you have any good suggestions? If computational cost is the issue, ideally, we could implement a fast atan2 here.
https://github.com/autowarefoundation/autoware.universe/blob/main/common/autoware_universe_utils/include/autoware/universe_utils/math/trigonometry.hpp

Reply:
Understand! I will implement the fast atan2 there!

Reply:
@yukkysaito  I move the fast atan2 function to trigonometry.cpp/hpp and provide unit test for the function.